data/graphs/backdoor/31a6a74c272af95f5223602e12ef1364527f584675445530eee060e635d2cafb.apk.top.dot

```diff
@@ -1,37 +0,0 @@
-strict digraph  {
```

Comment:
Why is this file deleted?

Reply:
this is moved for convenience of testing, I will move it back